documentation/src/test/java/org/hibernate/search/documentation/search/projection/ProjectionDslIT.java

```diff
@@ -205,6 +206,30 @@ public void field() {
 			);
 		} );
 
+		withinSearchSession( searchSession -> {
+			// tag::field-multiValued[]
+			List<List<String>> hits = searchSession.search( Book.class )
+					.select( f -> f.field( "authors.lastName", String.class ).multi() )
```

Comment:
Yes we need it, because it impacts the return type of the projection: for single-valued projections we return `String`, and for multi-valued projections we return `List<String>`.

If we were to infer that kind of things automatically, the projection would have to return `Object`  or to always return `List<String>`, which wouldn't be very nice...

Note that users who don't care can call `.multi()` even on single-valued field, and they'll always  get a `List<String>`.